### PR TITLE
NOISSUE - Add interval service

### PR DESCRIPTION
**Pull request by darkodraskovic**
Add interval service to trigger events on regular time intervals.

### CODE REVIEW

File: src/app/common/services/interval/interval.service.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+    this.callbacks.push(callback);
+
+    interval = interval || this.defaultInterval;
+
+    const id = window.setInterval(callback.bind(context), interval);
+    this.ids.push(id);
+  }
+
+  remove() {
```
Comment (manuio) [inline src/app/common/services/interval/interval.service.ts:25]: Use clear to be consistent with  `setInterval` and `cleartInterval`

File: src/app/pages/channels/details/channels.details.component.ts [hunk 2]
```diff
@@ -81,32 +83,38 @@ export class ChannelsDetailsComponent implements OnInit {
...
+    this.findDisconnectedThings();
+  }
 
+  findConnectedThings() {
+    this.channelsService.connectedThings(this.channel.id).subscribe(
+      (resp: any) => {
+        this.connectedThings = resp.things;
         this.getChannelMessages();
-      },
+        },
```
Comment (manuio) [inline src/app/pages/channels/details/channels.details.component.ts:103]: Remove this white space

File: src/app/pages/things/details/things.details.component.ts [hunk 3]
```diff
@@ -84,37 +86,53 @@ export class ThingsDetailsComponent implements OnInit {
...
+    this.connectedChans.forEach(chan => {
+      this.messagesService.getMessages(chan.id, this.thing.key, this.thing.id).subscribe(
+        (respMsg: any) => {
+          if (respMsg.messages) {
+            messages = messages.concat(respMsg.messages);
+            counter++;
+            if (counter === this.connectedChans.length - 1) {
+              this.messages = messages;
+            }
+          }
```
Comment (manuio) [inline src/app/pages/things/details/things.details.component.ts:129]: Please use:
```
respMsg.messages.forEach(msg=>
this.messages.push(msg);
);
```

**Review by manuio — CHANGES_REQUESTED** [review]

File: src/app/common/services/interval/interval.service.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+
+type Callback = () => void;
+
+@Injectable()
+export class IntervalService {
+  ids: number[] = [];
+  callbacks: Callback[] = [];
+  defaultInterval: number = 1000 * 5; // ms * sec
```
Comment (manuio) [inline src/app/common/services/interval/interval.service.ts:9]: If this is a const I would move it below `type Callback`

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by mteodor — APPROVED** [review]

**Pull request merged**